classes/cmake.yaml

```diff
@@ -38,10 +45,8 @@ buildScript: |
     set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
     set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)
     EOF
-        CMAKE_TOOLCHAIN_FILE="$PWD/toolchain.cmake"
-        BUILD_SHARED_LIBS=ON
-    else
-        BUILD_SHARED_LIBS=OFF
```

Comment:
Any reason why you removed this? The rationale is that all natively built libraries should be static because they are presumably used for host tools.

Reply:
What I missed here, is that the user can set the type manually inside the recipe.
making a library static or shared should not be limited by the system.
We need a mechanism to overwrite this.